proposals/low-level-struct-improvements.md

```diff
@@ -157,7 +156,7 @@ Next the rules for ref re-assignment need to be adjusted for the presence of `re
 
 > For a ref reassignment in the form ...
 > 1. `x.e1 = ref e2`: where `x` is *safe-to-escape* to *calling method* then `e2` must be *ref-safe-to-escape* to the *calling method*
-> 2. `e1 = ref e2`: where `e1` is a local or parameter, the *ref-safe-to-escape* of `e2` must be at least as wide a scope as the *ref-safe-to-escape* of `e1`.
+> 2. `e1 = ref e2`: where `e1` is a `ref` local or parameter then `e2` must have a *ref-safe-to-escape* at least as big as the *ref-safe-to-escape* of `e1`.
```

Comment:
No change here. Just tried to get the language consistent between the rules.

Reply:
My issue with using "big" here is that in the implementation, smaller numbers are used to represent "wider", also known as "bigger" scopes. :)

Reply:
It is unfortunate that the original implementation occurred before the spec and the spec didn't re-use the implementation language. When we look at cleaning up the ref saftey implementation in MQ we should also think a lot about getting the language in sync with the spec.